### DISCUSSION TITLE
ScreenToGif 2.40.1

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#1269] [Announcements]
# What's new?

• Bug fixes and translation updates.  
• Updated Dutch, Chinese, Polish and Turkish localizations.  

### Fixed:

♦ Fixed issue of wrong window padding when maximized.  
♦ Fixed crash when trying to run the app installed with the MSIX installer.      

### More info:

[Known issues](https://github.com/NickeManarin/ScreenToGif/wiki/Help-▪-Known-issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/NickeManarin/ScreenToGif/releases/tag/2.40.1'>ScreenToGif 2.40.1</a>.</em>

**Comment by Yihsien124** [comment]
Hi, Alt + Arrow Left/Right commands were not working.
Is there any configuration that can restore the functionality of this command?
Thank you.

**Reply by NickeManarin** [reply]
I'm going to check what's happening and fix it.
